src/databricks/labs/ucx/queries/views/object_estimates.sql

```diff
@@ -0,0 +1,35 @@
+WITH estimated_hours AS (
+  SELECT * FROM (VALUES
+    ('jobs', 6, 'not supported DBR:'), -- Upgrade cluster to use a DBR 11.3+ LTS version
```

Comment:
It's out of the scope of the estimate, because it boils down to the code that is submitted to the cluster that dictates the time to upgrade, not the cluster itself.

Reply:
sgtm, document it in the assumption then